docs/api.rst

```diff
@@ -206,15 +206,17 @@ For multiple parallel queries logged in to the same object, you could do:
 
 .. code-block:: python
 
-    from astroquery import module
-
+    from astroquery.module import QueryClass
+    
     QC = QueryClass(login_information)
 
     results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])
```

Comment:
While you are at it, would you mind applying some PEP8 to this example? It is a bit hard to read.